src/mumble/main.cpp

```diff
@@ -221,227 +223,189 @@ int main(int argc, char **argv) {
-								   "  --dump-input-streams\n"
-								   "                Dump PCM streams at various parts of the input chain\n"
-								   "                (useful for debugging purposes)\n"
-								   "                - raw microphone input\n"
-								   "                - speaker readback for echo cancelling\n"
-								   "                - processed microphone input\n"
-								   "  --print-echocancel-queue\n"
-								   "                Print on stdout the echo cancellation queue state\n"
-								   "                (useful for debugging purposes)\n"
-								   "  --translation-dir <dir>\n"
-								   "                Specifies an additional translation directory <dir>\n"
-								   "                in which Mumble will search for translation files that\n"
-								   "                overwrite the bundled ones\n"
-								   "                Directories added this way have higher priority than\n"
-								   "                the default locations used otherwise\n"
-								   "  --print-translation-dirs\n"
-								   "                Print out the paths in which Mumble will search for\n"
-								   "                translation files that overwrite the bundled ones.\n"
-								   "                (Useful for translators testing their translations)\n"
-								   "  --locale <locale>\n"
-								   "                Overwrite the locale in Mumble's settings with a\n"
-								   "                locale that corresponds to the given locale string.\n"
-								   "                If the format is invalid, Mumble will error.\n"
-								   "                Otherwise the locale will be permanently saved to\n"
-								   "                Mumble's settings."
-								   "\n");
-				QString rpcHelpBanner = MainWindow::tr("Remote controlling Mumble:\n"
-													   "\n");
-				QString rpcHelpMessage =
-					MainWindow::tr("Usage: mumble rpc <action> [options]\n"
-								   "\n"
-								   "It is possible to remote control a running instance of Mumble by using\n"
-								   "the 'mumble rpc' command.\n"
-								   "\n"
-								   "Valid actions are:\n"
-								   "  mute\n"
-								   "                Mute self\n"
-								   "  unmute\n"
-								   "                Unmute self\n"
-								   "  togglemute\n"
-								   "                Toggle self-mute status\n"
-								   "  deaf\n"
-								   "                Deafen self\n"
-								   "  undeaf\n"
-								   "                Undeafen self\n"
-								   "  toggledeaf\n"
-								   "                Toggle self-deafen status\n"
-								   "  starttalking\n"
-								   "                Start talking\n"
-								   "  stoptalking\n"
-								   "                Stop talking\n"
-								   "\n");
-
-				QString helpOutput = helpMessage + rpcHelpBanner + rpcHelpMessage;
-				if (bRpcMode) {
-					helpOutput = rpcHelpMessage;
-				}
 
+	CLI::App app;
+
+	std::string custom_jack_client_name;
+	bool print_license{ false };
```

Comment:
```suggestion
	bool print_license = false;
```
To stay consistent with other code in Mumble, initialization of literals should be done by explicit assignment